### DISCUSSION TITLE
coding for data science - UCB MIDS or Georgia Tech OMSA

### DISCUSSION THREAD

**Discussion by arashmehrabiparsa** [codewars/codewars.com#3073] [General]
im a little scared of coding in python and i feel like i need more ways to apply what i have read in books about:

- self-attention in transformer networks
- convex optimization
- XGBoost
-  deep learning
- diffusion model
- Applied Natural Language Processing
-  CNN, CV, NLP
- Markov decision processes
- reinforcement learning
- Monte Carlo tree search (monte calro methods), 
- recurrent nueral networks, 
- gradient descent backpropagation,
- q-learning, 
- KNN, 
- ARIMA, 
- GARCH, 
- PCA, 
- optimization, 
- k-means, 
- SVM,
-  classification, 
- poisson regression (Poisson Generative Flow Model) 
into actual python code. Now I am trying codewars to see if I gain some skills but codewars isn't necessarily prepping for data science right?
